File: 0476-number-complement/0476-number-complement.java
class Solution {
    public int findComplement(int num) {
        String binaryString = Integer.toBinaryString(num);
        char [] binArray=binaryString.toCharArray();
        String s="";
        for(int i=0;i<binArray.length;i++){
            if(binArray[i]=='1'){
                s+='0';
            }
            else{
                s+='1';
            }
        }
        int number=Integer.parseInt(s,2);

        return number;
    }
}

File: 0476-number-complement/0476-number-complement.py
class Solution(object):
    def findComplement(self, num):
        bin_str=bin(num)[2:]
        str=""
        for i in range(len(bin_str)):
            if bin_str[i]=='0':
                str+='1'
            else:
                str+='0'    
        number=int(str,2)
        return number
        """
        :type num: int
        :rtype: int
        """
        